keystored/scripts/ssh-key-import.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+shopt -s failglob
+
+local_path=$(dirname $0)
+
+: ${SYSREPOCFG:=sysrepocfg}
+: ${CHMOD:=chmod}
+: ${OPENSSL:=openssl}
+: ${STOCK_KEY_CONFIG:=$local_path/../stock_key_config.xml}
+: ${KEYSTORED_KEYS_DIR:=/etc/keystored/keys}
+
+if [ -n "$($SYSREPOCFG -d startup --export ietf-keystore)" ]; then
+    exit 0
+fi
+
+if [ -f /etc/ssh/ssh_host_rsa_key ]; then
```

Comment:
This is wrong because it checks that file for existence, not readability. Installing as non-root fails.

It also partially undoes changes I made there recently which let me pre-provision the configuration without actually having the key on-disk. I'll fix that one in a followup patch.

Reply:
Well, I consider these installation scripts so you should run them as `root`. Still, there is copying performed right after it so you are right that we should check for read permissions.

Reply:
Sorry for the mistakes that were made, and thanks a lot for taking care of the fixes.